cmd/minishift/cmd/root.go

```diff
@@ -71,6 +73,10 @@ var RootCmd = &cobra.Command{
 	Short: "Minishift is a tool for application development in local OpenShift clusters.",
 	Long:  `Minishift is a command-line tool that provisions and manages single-node OpenShift clusters optimized for development workflows.`,
 	PersistentPreRun: func(cmd *cobra.Command, args []string) {
+		if minishiftStrings.Contains(cmdIgnoreCreatingMinishiftHomeDir, cmd.Name()) && cmd.Parent().Name() == constants.DefaultMachineName {
+			return
```

Comment:
@budhrg I am not sure why we need `&& cmd.Parent().Name() == constants.DefaultMachineName` check?  Can you please explain?

Reply:
@LalatenduMohanty Because, cmd.Name() for "minishift openshift version" is coming as "version" and then it is getting returned from here itself without created need files for `minishift openshift version`.

does cmd.Parent().Name() == "minishift" works for you?

Reply:
Check the errors in integration logs of cmd-openshift.feature https://ci.centos.org/job/minishift-pr/4070/console